lib/tasks/web.rake

```diff
@@ -74,6 +74,33 @@ namespace :slack do
           'bin/commands.rb',
           commands_template.result(files: data.keys.map { |key| key.tr('.', '_') })
         )
+
+        ### Errors
+
+        # Use the official spec to pull a list of errors and create classes for each
+
+        spec = JSON.load(open('https://raw.githubusercontent.com/slackapi/slack-api-specs/master/web-api/slack_web_openapi_v2.json'))
+        known_typos = %w[account_inactiv token_revokedno_permission]
+
+        errors = spec['paths'].map do |_, path|
+          path.map do |_, v|
+            v['responses'].map do |_, response|
+              response.dig('schema', 'properties', 'error', 'enum')
```

Comment:
Is it OK to use `dig`? Can we assume anyone running this is on a new enough ruby? I'm not sure all the versions supported by the gem are new enough for `dig`.

Reply:
Actually it looks like we test back to 2.3, which is when `dig` was added.

Reply:
I suppose it's fine.